src/cli.ts

```diff
@@ -190,7 +239,7 @@ npx autoblocks testing exec -- echo "Hello, world!
       });
 
       // Send error to Autoblocks
-      await tracer.sendEvent('cli.error', {
+      tracer.sendEvent('cli.error', {
```

Comment:
Fire and forget 🔥 nice.

Reply:
i actually probably need to call flush here since i don't think process.exit will wait for event loop to be empty?